tests/unit/BootstrapCest.php

```diff
@@ -10,14 +10,20 @@ class BootstrapCest
 {
     public function checkBootstrap(CliTester $I)
     {
+        /**
+         * As current request is inside CLI
+         * There are no REQUEST URI
+         */
+        $_SERVER['REQUEST_URI'] = '/';
```

Comment:
Could you please move this to the bootstrap or/and module initialization

Reply:
Specified as action point in #23